<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog111</class>
 <widget class="QDialog" name="Dialog111">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>896</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="Prefix_List">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="autoExpandDelay">
    <number>11</number>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="GlovesButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Gloves</string>
   </property>
  </widget>
  <widget class="QTreeView" name="Suffix_List">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>20</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
